cmd/kubectx/list.go

```diff
@@ -19,10 +18,6 @@ func (_ ListOp) Run(stdout, stderr io.Writer) error {
 	kc := new(kubeconfig.Kubeconfig).WithLoader(kubeconfig.DefaultLoader)
 	defer kc.Close()
 	if err := kc.Parse(); err != nil {
-		if cmdutil.IsNotFoundErr(err) {
-			printer.Warning(stderr, "kubeconfig file not found")
-			return nil
-		}
```

Comment:
why was this deleted?

Reply:
[Found and removed duplicate code for "Kubeconfig not found" in lists.go, which is already handled in fzf.go. Also, it seems like fzf.go will always supersede when there is an error as such.](https://github.com/ahmetb/kubectx/blob/51c61b862dcbf2293d5c037423356523e7df9e96/cmd/kubectx/fzf.go#L28)

Reply:
but list.go#Run and fzf.go#Run are for different scenarios.
fzf.go runs if fzf is installed, list.go runs otherwise.

Reply:
oh! I understand now. apologies